src/modules/analytics/services/global.rewards.service.ts

```diff
@@ -0,0 +1,266 @@
+        private readonly remoteConfig: RemoteConfigGetterService,
+        private readonly farmFactory: FarmFactoryService,
+        private readonly weekTimekeepingAbi: WeekTimekeepingAbiService,
+        private readonly tokenCompute: TokenComputeService,
+        private readonly stakingService: StakingService,
+    ) {}
+
+    async getGlobalRewards(weekOffset: number): Promise<GlobalRewardsModel> {
+        return new GlobalRewardsModel({});
+    }
+
+    @GetOrSetCache({
+        baseKey: 'analytics',
+        remoteTtl: Constants.oneHour() * 6,
+        localTtl: Constants.oneHour() * 4,
+    })
+    async feesCollectorRewards(
+        weekOffset: number,
+    ): Promise<FeesCollectorGlobalRewards> {
+        try {
+            return await this.processFeesCollectorRewards(weekOffset);
+        } catch (error) {
+            this.logger.error(
+                `Error getting fees collector rewards: ${error.message}`,
+            );
+            throw new Error('Error getting fees collector rewards');
+        }
+    }
+
+    @GetOrSetCache({
+        baseKey: 'analytics',
+        remoteTtl: Constants.oneHour() * 6,
+        localTtl: Constants.oneHour() * 4,
+    })
+    async farmRewards(weekOffset: number): Promise<FarmsGlobalRewards[]> {
+        try {
+            return await this.processFarmsRewards(weekOffset);
+        } catch (error) {
+            this.logger.error(`Error getting farms rewards: ${error.message}`);
+            throw new Error('Error getting farms rewards');
+        }
+    }
+
+    @GetOrSetCache({
+        baseKey: 'analytics',
+        remoteTtl: Constants.oneHour() * 6,
+        localTtl: Constants.oneHour() * 4,
+    })
+    async stakingRewards(weekOffset: number): Promise<StakingGlobalRewards[]> {
+        try {
+            return await this.processStakingRewards(weekOffset);
+        } catch (error) {
+            this.logger.error(
+                `Error getting staking rewards: ${error.message}`,
+            );
+            throw new Error('Error getting staking rewards');
+        }
+    }
+
+    private async processFeesCollectorRewards(
+        weekOffset: number,
+    ): Promise<FeesCollectorGlobalRewards> {
+        let energyRewardsUSD = new BigNumber(0);
+        const feesCollectorAddress = scAddress.feesCollector;
+
+        const currentWeek =
+            await this.weeklyRewardsSplitting.lastGlobalUpdateWeek(
+                feesCollectorAddress,
+            );
+
+        const targetWeek = Math.max(0, currentWeek - weekOffset);
+
+        const weekRewardsUSD =
+            await this.weeklyRewardsSplittingCompute.totalRewardsForWeekUSD(
+                feesCollectorAddress,
+                targetWeek,
+            );
+
+        if (weekRewardsUSD && weekRewardsUSD !== '0') {
+            energyRewardsUSD = energyRewardsUSD.plus(
+                new BigNumber(weekRewardsUSD),
+            );
+        }
+
+        const totalRewardsUSD = energyRewardsUSD.dividedBy(0.6);
```

Comment:
All feesCollector rewards are energy rewards. No need to get a part from the rewards